### PR TITLE
DUMMY PR - IGNORE

### PR DESCRIPTION
**Pull request by heplesser**

### CODE REVIEW

File: testsuite/pytests/sli2py_recording/test_multimeter_stepping.py [hunk 1]
```diff
@@ -88,10 +89,14 @@ def build_net(model):
     """
 
     nest.ResetKernel()
-
+    if model.startswith("eprop_"):
+        nest.resolution = 1.0
+        nest.eprop_update_interval = 1.0
+        nest.eprop_learning_window = 1.0
```
Comment (heplesser) [inline testsuite/pytests/sli2py_recording/test_multimeter_stepping.py:95]:
```suggestion
        nest.resolution = default_delay
        nest.eprop_update_interval = nest.resolution
        nest.eprop_learning_window = nest.resolution
```

Comment (heplesser) [reply testsuite/pytests/sli2py_recording/test_multimeter_stepping.py:95]: Does one need to set the last two at all here?

File: testsuite/pytests/sli2py_recording/test_multimeter_stepping.py [hunk 2]
```diff
@@ -88,10 +89,14 @@ def build_net(model):
...
+        nest.eprop_reset_neurons_on_update = True
     nrn = nest.Create(model)
     pg = nest.Create("poisson_generator", params={"rate": 1e4})
-    mm = nest.Create("multimeter", {"interval": 0.1, "record_from": nrn.recordables})
+    mm = nest.Create("multimeter", {"interval": max(0.1, nest.resolution), "record_from": nrn.recordables})
```
Comment (heplesser) [inline testsuite/pytests/sli2py_recording/test_multimeter_stepping.py:99]: Why max? Why 0.1?

File: testsuite/pytests/sli2py_recording/test_multimeter_stepping.py (hunk 1 above)
Comment (akorgor) [reply testsuite/pytests/sli2py_recording/test_multimeter_stepping.py:95]: No, they can be deleted.

**Pull request closed without merging**